aomi/helpers.py

```diff
@@ -249,11 +249,12 @@ def dict_unicodeize(some_dict):
     return some_dict
 
 
-def diff_dict(dict1, dict2):
+def diff_dict(dict1, dict2, ignore_missing=False):
```

Comment:
I needed something similar for the CloudOS health checker, and used the DeepDiff library. YMMV.